src/modules/recipe/helloExpressionsTrack/helloExpressionsTrack.html

```diff
@@ -0,0 +1,25 @@
+<template>
+    <ui-card title="HelloExpressionsTrack">
+        <div>
+            <ui-input
+                name="firstName"
+                label="First Name"
+                onchange={handleChange}
+            ></ui-input>
+            <ui-input
+                name="lastName"
+                label="Last Name"
+                onchange={handleChange}
+            ></ui-input>
+            <p class="margin-top-medium">
+                Uppercased Full Name: {uppercasedFullName}
+            </p>
+        </div>
+
+        <recipe-view-source source="recipe/helloExpressionsTrack" slot="footer">
```

Comment:
`leverages the Track Decorator` => `leverages the @track decorator`.

Also I feel that this help should nail the difference (because of object fields).

Reply:
Agreed, let's highlight the difference so that people don't have to read the source code to spot the difference.